benchmark/run_all.dart

```diff
@@ -24,27 +26,46 @@ List<int> throughputParserHelper(String aotThroughputStdout) {
 }
 
 void main(List<String> args) async {
-  var url = 'https://example.com';
-  var throughputPrallelLimit = 10;
-  var duration = const Duration(seconds: 1);
-  if (args.isNotEmpty) {
-    url = args[0];
-  }
-  if (args.length > 1) {
-    throughputPrallelLimit = int.parse(args[1]).toInt();
+  final parser = ArgParser();
+  parser
+    ..addOption('url',
+        abbr: 'u',
+        help: 'The server to ping for running this benchmark.',
+        defaultsTo: 'https://example.com')
+    ..addOption('limit',
+        abbr: 'l',
+        help: 'Limits the maximum number of parallel requests to 2^N where N'
+            ' is provided through this option.',
+        defaultsTo: '10')
+    ..addOption('time',
+        abbr: 't',
+        help: 'Maximum second(s) the benchmark should wait for each request.',
+        defaultsTo: '1')
+    ..addFlag('help',
+        abbr: 'h', negatable: false, help: 'Print this usage information.');
+  final arguments = parser.parse(args);
+  if (arguments.wasParsed('help')) {
+    print(parser.usage);
+    return;
   }
-  if (args.length > 2) {
-    duration = Duration(seconds: int.parse(args[2]));
+  if (arguments.rest.isNotEmpty) {
+    print(parser.usage);
+    throw ArgumentError();
   }
+  final url = arguments['url'] as String;
+  final throughputPrallelLimit = int.parse(arguments['limit'] as String);
+  final duration = Duration(seconds: int.parse(arguments['time'] as String));
```

Comment:
Does this work with half a second?

Reply:
No. `Duration`'s `seconds` param takes `int`.

Reply:
Maybe we should do milliseconds instead then, or convert a double manually. Because otherwise we can't test shorter durations.

Reply:
Let's go with milliseconds then 😀

Reply:
Got this result under short duration. Cronet is so much faster.

Latency Test Results
| Mode          | package:cronet | dart:io        |
| :-----------: |:-------------: | :------------: |
| JIT           | 247.333 ms   | 403.000 ms   |
| AOT           | 240.444 ms   | 373.000 ms   |

Throughput Test Results (Duration: 500 ms)
| Mode          | package:cronet  | dart:io        |
| :-----------: |:--------------: | :-----------:  |
| JIT           | 30 out of 1024  | 0 out of 0  |
| AOT           | 26 out of 1024 | 0 out of 0 |